config/test.exs

```diff
@@ -0,0 +1,2 @@
+use Mix.Config
+config :docker_api, :host, "127.0.0.1:2376"
```

Comment:
Lets make this a env var?

`config :docker_api, :host, Sytem.get_env("DOCKER_API_HOST") `